Makefile

```diff
@@ -1,6 +1,3 @@
-include .env
```

Comment:
This was made intentionally. Manually exporting some variables (and finding their values) each time you want to hack into the library is a real pain. This things makes it easy for developpers